mevolib/cluster/Genes.py

```diff
@@ -125,215 +85,171 @@
                     '5\'UTR': ['gene', 'standard_name'] }
 
 
-#-------------------------------------------------------------------------------
-
-def _normalization ( record, refseq_record, alignment_bin ) :
-    """
-    Normalization of the input sequence with the reference sequence. The
-    normalization consists on aligning both sequences and removing those sites
-    where a gap has been introduced in the reference sequence, making the
-    features of the reference sequence applicable to the new sequence. It
-    returns the new sequence and the reference sequence's features.
+def _normalization(record: Bio.SeqRecord, refseq_record: Bio.SeqRecord, 
+                   alignment_bin: str) -> tuple(Bio.Seq, List):
+    """Normalization of the input sequence with the reference sequence. The normalization consists on aligning 
+    both sequences and removing those sites where a gap has been introduced in the reference sequence, making 
+    the features of the reference sequence applicable to the new sequence. It returns the new sequence and the 
+    reference sequence's features.
```

Comment:
Short definition in one line and, if required, a longer one in the next "block":
```suggestion
    """Normalizes the input sequence with the reference sequence.
    
    The normalization consists on aligning both sequences and removing those sites where a gap has
    been introduced in the reference sequence, making the features of the reference sequence
    applicable to the new sequence. It returns the new sequence and the reference sequence's features.
```